src/pkg/path/elements.go

```diff
@@ -86,3 +105,16 @@ func (el Elements) String() string {
 func (el Elements) PlainString() string {
 	return join(el)
 }
+
+// ---------------------------------------------------------------------------
+// helpers
+// ---------------------------------------------------------------------------
+
+// LoggableDir takes in a path reference (of any structure) and conceals any
+// non-standard elements (ids, filenames, foldernames, etc).
+func LoggableDir(ref string) string {
+	elems := Split(ref)
```

Comment:
may want to do some extra trimming on this and/or ensure this is only used for safe call paths. Split has the note that it can't handle unescaped trailing `/`

Reply:
done